TLM/TLM/Manager/Impl/VehicleBehaviorManager.cs

```diff
@@ -347,7 +347,7 @@ public class VehicleBehaviorManager : AbstractCustomManager, IVehicleBehaviorMan
 
                                     // If you can turn preferred and you're not going straight, continue
                                     SegmentGeometry currentSegGeo = SegmentGeometry.Get(uCurrentSegment);
-                                    SegmentEndGeometry currentSegEndGeo = currentSegGeo.EndNodeGeometry;
+                                    SegmentEndGeometry currentSegEndGeo = currentSegGeo.GetEnd(true);
```

Comment:
Am I seeing correctly? currentSegGeo.GetEnd(true)? 
What is that 'true' and more interesting how it's even casted to type ushort 😆

Reply:
GetEnd asks for a bool of startNode. I am confused?

Reply:
`(+1 overload)` 😉